### ISSUE TITLE
Failed to connect to ftp://xfer13.crg.eu:221

### ISSUE THREAD

**Issue by SergiAguilo** [EGA-archive/beacon2-ri-tools#10]
Hi,

I'm unable to download the files from ftp://xfer13.crg.eu:221 needed for the containerized or non-containerized tool.

```
> wget ftp://xfer13.crg.eu:221/beacon2_data.md5
--2022-08-03 12:32:35--  ftp://xfer13.crg.eu:221/beacon2_data.md5
           => ‘beacon2_data.md5’
Resolving xfer13.crg.eu (xfer13.crg.eu)... 84.88.52.84
Connecting to xfer13.crg.eu (xfer13.crg.eu)|84.88.52.84|:221... failed: No route to host.

```

Thanks!

**Comment by mrueda** [comment]
Hi @SergiAguilo, the files are stored in a server located at the BSC. The BSC is under maintenance until tmw so please give it a couple of days.

Thanks,

Manu

**Comment by gemaroldan** [comment]
Hi:
   I have the same problem and I can't download the files either.
   When will the server be up?

My best wishes

**Comment by mrueda** [comment]
Hi Gema,

Apologies for that. We're aware of the issue. The IT folks at CRG are working on a solution. 
I will let you know when the ftp is accessible again.

Thx,

Manu

**Comment by gemaroldan** [comment]
Ok thanks for the information. I will wait for you to notify me.

**Comment by mrueda** [comment]
Hi @gemaroldan,

The ftp issue should be fixed by now #12. 

Best,

m

**Comment by gemaroldan** [comment]
I can now download files. 

Thank you
